### ISSUE TITLE
indexOf: fromIndex

### ISSUE THREAD

**Issue by petetalksweb** [sdras/array-explorer#81]
I feel like `indexOf`'s optional second parameter: `fromIndex`, is an unsung hero. 

It has saved me many times from having to `slice` an array before calling `indexOf`.

Would you consider a PR with an example using `fromIndex`?

**Comment by sdras** [comment]
absolutely! it would be great if you could explain/show it in a way that didn't cloud newcomers understanding of indexOf.

The other thing is updating the languages. We could either drop it into google translate or just leave it different, I'm open to either.

I look forward to it!

**Comment by amanchauhan98** [comment]
Indexof ? I can't understand!! plaese somebody elobrate this.
